src/main/resources/application.yml

```diff
@@ -15,12 +19,14 @@ spring:
         - dev
       test:
         - stdout
-        - dev
+        - test
 
   jackson:
     date-format: "yyyy-MM-dd HH:mm:ss"
     serialization:
       WRITE_DATES_AS_TIMESTAMPS: false
+    default-property-inclusion: non_null
+##    default-property-inclusion: non_empty
```

Comment:
Not sure if you left this on purpose?

Reply:
Its related to a decision I need to make with the client - I will update the code and add to the ticket instead.